{
  this.incrementCounter("schema_exists");
  LOG.info("schema_exists(" + dbName + ","+ tableName+ ")");
  try {
    return new DB(dbName,conf_).tableExists(tableName);
  }
 catch (  MetaException e) {
    LOG.error("Got exception in schema_exists: " + e.getMessage());
    this.incrementCounter("exceptions");
    throw e;
  }
catch (  UnknownDBException e) {
    LOG.error("Got exception in schema_exists: " + e.getMessage());
    this.incrementCounter("exceptions");
    throw e;
  }
catch (  RuntimeException e) {
    LOG.fatal("schema_exists got a runtime exception: " + e.getMessage());
    MetaStoreUtils.printStackTrace(e);
    throw new MetaException("schema_exists had an internal Java RuntimeException: " + e.getMessage());
  }
}
